### ISSUE TITLE
Final Report Peer Review (By zw243)

### ISSUE THREAD

**Issue by wangzilongri** [Tongcheng/4741_TrendTrade#9]
# TrendTrade 
# Final Report Review
### Abstract

The abstract is very well written and provides very good motivation for why 
this project is an interesting one. It is highly interesting to consider how
often a stock is searched for on Google will affect its prices. This grabs
the reader's attention immediately.

### Data Gathering

This section documented the various technical issues with obtaining clean data
in clear detail, such as APIs restricting the number of queries in a certain 
time period.

For typing (2), due to the poor way it was phrased, it may mislead readers to
believe that it was a type conversion/casting issue, and not typographical.
This is a minor issue and should not have been included in this section.

In addition, for those who are unfamilliar with trading terms, the author should
first explain what "black swans" and "white swans" are.

### Cleanup + Models

For the linear regression, it is very unclear what the author is using for his
linear transform function and loss function. In additon, if he wants to list
the weights for the coefficients, a table format would be much clearer.

For the Quantile Regression section, there seems to be clear trend amongst
all the plots, but as the author has not explained what they mean, (not even
a legend was available), the section has been wasted information wise.

### Backtesting and Summary

This section serves as some form of a "evaluation set" for the author's chosen
model(s). However, the author seems to have veered off track, as there is no
longer any mention of Google Trend data, and the summary (which is supposed
to summarise how well his models work) turned out to be a summary of what he
did, which is not very helpful.

### Other comments

English is probably not the author's native language, so mistakes in grammar/
idiomatic usage should be overlooked. However, the information is stil not
presented in a clear manner and often leaves the reader wondering what is the
plot showing.

In addition, using a markdown document circumvents the 8 page limit that many
other teams had to abide to.
